GPU/GPUTracking/kernels.cmake

```diff
@@ -110,6 +110,12 @@ o2_gpu_add_kernel("GPUTPCCFPeakFinder"                                "= TPCCLUS
 o2_gpu_add_kernel("GPUTPCCFNoiseSuppression, noiseSuppression"        "= TPCCLUSTERFINDER"                                    LB      single)
 o2_gpu_add_kernel("GPUTPCCFNoiseSuppression, updatePeaks"             "= TPCCLUSTERFINDER"                                    LB      single)
 o2_gpu_add_kernel("GPUTPCCFDeconvolution"                             "= TPCCLUSTERFINDER"                                    LB      single)
+o2_gpu_add_kernel("GPUTPCNNClusterizer, runCfClusterizer"             "= TPCCLUSTERFINDER"                                    LB      single int8_t dtype int8_t onlyMC uint batchStart)
```

Comment:
This will try to compile the kernels for all GPU backends, so it will certainly fail in the FullCI when compiling for CUDA/ROCm/OpenCL.
If I understand correctly, you call ONNX here, so anyway you don't want to call that from the GPU. So there is no need to use the kernel infrastructure here.
Why don't you create a separate class and call it directly from GPUChainTrackingClusterizer?